index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My conference capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
+    crossorigin="anonymous">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="icon" id="bars">
+                <i class="fas fa-bars"></i>
+            </div>
```

Comment:
- [ ] There should be a navbar menu on the mobile screen but it doesn't appear on yours.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/128993244-c8495a20-725d-4f9e-9227-3a024ac76dce.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/128993017-a9ca814e-4716-4a52-87fc-e15f81312683.png' height='250px'> |